lib/src/third_party/boringssl/ffigen.yaml

```diff
@@ -1,17 +1,34 @@
 name: BoringSsl
 description: 'Bindings to BoringSSL.'
+language: c
 output: 'lib/src/third_party/boringssl/generated_bindings.dart'
 headers:
   entry-points:
-    - 'third_party/boringssl/src/include/openssl/*.h'
+    # bytestring.h MUST be the first header, otherwise structs are just left
+    # as opaque.
+    - 'third_party/boringssl/src/include/openssl/bytestring.h'
```

Comment:
@dcharkes, @mannprerak2, I have a scenario here where `base.h` forward declares `CBS` as:

```c
// from base.h
typedef struct cbs_st CBS;
```

But in `bytestring.h` we have the full `cbs_st` declaration:
```c
// from bytestring.h
struct cbs_st {
  const uint8_t *data;
  size_t len;
};
```

If `base.h` is read first by ffigen, then I get:
```dart
class cbs_st extends ffi.Opaque {}
```

But if `bytestring.h` is read first, then I get:
```dart
class cbs_st extends ffi.Struct {
  external ffi.Pointer<ffi.Uint8> data;

  @ffi.Size()
  external int len;
}
```
(which is what I need in order to be able to allocate and instance of the struct)

--------------------------------

All the other headers include `base.h` (or at-least many of them do). So my options here are:
 * Put `bytestring.h` at the top of the list.
 * Create my own `amalgamation.h` which includes all the headers I need.

I'm wondering if this is working as intended, or if it's a bug? I suspect this pattern is common in C-libraries.